pages/paths/[id].jsx

```diff
@@ -229,3 +183,58 @@ const PathDetailPage = () => {
 };
 
 export default PathDetailPage;
+
+export async function getServerSideProps(context) {
+  const { id } = context.params;
+  let pathData = null;
+  let markdownContent = null;
+
+  try {
+    const dataFilePath = path.resolve('components/Paths/data.json');
+    const fileContents = fs.readFileSync(dataFilePath, 'utf8');
+    const paths = JSON.parse(fileContents);
+
+    pathData = paths.find((currentPath) => currentPath.id === id);
+
+    if (pathData?.markdownPath) {
+      const markdownPath = path.resolve('public/', pathData.markdownPath);
```

Comment:
for me all paths are empty, is there some issue?
<img width="1434" alt="image" src="https://github.com/user-attachments/assets/785c7204-8b75-4684-8f31-eec9ccaa757d" />